src/mqt/bench/benchmark_generation.py

```diff
@@ -2,26 +2,21 @@
 
 from __future__ import annotations
 
-import argparse
-import json
 import signal
 import sys
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Literal, TypedDict, overload
 from warnings import warn
 
-from joblib import Parallel, delayed
```

Comment:
do we still need the `joblib` dependency if this is removed here?
(if not, please remove it and update the lock file with `uv lock`)